Flaky Test "UseSpa_KillsRds_WhenAppIsStopped"

Test failed with:

> npm ERR! code EPIPE\r\nnpm ERR! syscall write\r\nnpm ERR! errno EPIPE\r\n\r\nExpected: True\r\nActual:   False

>    at Microsoft.AspNetCore.SpaServices.Extensions.Tests.SpaServicesExtensionsTests.AssertNoErrors() in /_/src/Middleware/SpaServices.Extensions/test/SpaServicesExtensionsTests.cs:line 162
   at Microsoft.AspNetCore.SpaServices.Extensions.Tests.SpaServicesExtensionsTests.Assert_NpmKilled_WhenAppIsStopped(IHostApplicationLifetime applicationLifetime, NpmStartedDiagnosticListener listener) in /_/src/Middleware/SpaServices.Extensions/test/SpaServicesExtensionsTests.cs:line 97
   at Microsoft.AspNetCore.SpaServices.Extensions.Tests.SpaServicesExtensionsTests.UseSpa_KillsRds_WhenAppIsStopped() in /_/src/Middleware/SpaServices.Extensions/test/SpaServicesExtensionsTests.cs:line 57
--- End of stack trace from previous location ---

https://github.com/dotnet/aspnetcore/blob/db3d23b3af6c2ade6635faac35cbaff4a94694c1/src/Middleware/SpaServices.Extensions/test/SpaServicesExtensionsTests.cs#L42

CC @AndriySvyryd @ryanbrandenburg

--- a/src/Middleware/SpaServices.Extensions/test/SpaServicesExtensionsTests.cs
+++ b/src/Middleware/SpaServices.Extensions/test/SpaServicesExtensionsTests.cs
@@ -39,7 +39,6 @@ namespace Microsoft.AspNetCore.SpaServices.Extensions.Tests
         }
 
         [Fact]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/27552")]
         public async Task UseSpa_KillsRds_WhenAppIsStopped()
         {
             var serviceProvider = GetServiceProvider(s => s.RootPath = "/");
@@ -80,7 +79,7 @@ namespace Microsoft.AspNetCore.SpaServices.Extensions.Tests
         private async Task Assert_NpmKilled_WhenAppIsStopped(IHostApplicationLifetime applicationLifetime, NpmStartedDiagnosticListener listener)
         {
             // Give node a moment to start up
-            await Task.WhenAny(listener.NpmStarted, Task.Delay(TimeSpan.FromSeconds(30)));
+            await Task.WhenAny(listener.NpmStarted, Task.Delay(TimeSpan.FromSeconds(45)));
 
             Process npmProcess = null;
             var npmExitEvent = new ManualResetEventSlim();
@@ -98,7 +97,7 @@ namespace Microsoft.AspNetCore.SpaServices.Extensions.Tests
             AssertNoErrors();
             Assert.True(listener.NpmStarted.IsCompleted, "npm wasn't launched");
 
-            npmExitEvent.Wait(TimeSpan.FromSeconds(30));
+            npmExitEvent.Wait(TimeSpan.FromSeconds(45));
             Assert.True(npmProcess.HasExited, "npm wasn't killed");
         }
 

